AndroidWeek1/app/src/main/java/org/sopt/androidweek/home/FollowerViewModel.kt

```diff
@@ -1,28 +1,39 @@
 package org.sopt.androidweek.home
 
+import android.util.Log
 import androidx.lifecycle.LiveData
 import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import kotlinx.coroutines.*
+import org.sopt.androidweek.repository.network.GitCreator
+import java.lang.Exception
 
 class FollowerViewModel : ViewModel() {
     private val _followerList = MutableLiveData<List<FollowerDao>>()
     val followerList: LiveData<List<FollowerDao>>
         get() = _followerList
 
     init {
-        _followerList.value = loadFollower()
+        loadFollowerApi()
     }
 
-
-    private fun loadFollower() : List<FollowerDao>{
-        val contentStr =
-            "가나다라마바사아자차카타파하가나다라마바사아자차카타파하" +
-                    "가나다라마바사아자차카타파하가나다라마바사아자차카타"
-        return (1..9).map { i ->
-            FollowerDao(
-                "이동기$i",
-                contentStr+ contentStr
-            )
+    private fun loadFollowerApi() {
+        var list = emptyList<FollowerDao>()
+        viewModelScope.launch {
+            try {
+                val copyList = mutableListOf<FollowerDao>()
+                list = GitCreator.gitService.getGitHubFollowers("rkdmf1026")
+                for (follower in list)
+                    copyList.add(
+                        FollowerDao(
+                            follower.name,
+                            GitCreator.gitService.getUserInfo(follower.name).introduction
+                        )
+                    )
+                _followerList.postValue(copyList)
```

Comment:
```suggestion
                _followerList.postValue(list.toList())
```

- toList만 사용하셔도 리스트 주소의 복사가 가능합니다.
- list라는 이름만 봤을 때 어떤 데이터인지 파악이 힘듭니다. 변수명에 이 데이터의 의도를 명시해주시는 것이 좋을 것 같습니다.